src/mongo_module/models/adaptiveLearning.ts

```diff
@@ -0,0 +1,28 @@
+import mongoose from "mongoose";
+
+// Define the User schema
+const adaptive_learning = new mongoose.Schema({
+    udise_code : {
+        type: String,
+        required: true,
+        index: true
+    },
+    school_name: {
+        type: String,
+        required: true,
+        index: true
+    },
+    createdAt: {
+        type: Date,
+        default: Date.now
+    },
+    updatedAt: {
+        type: Date,
+        default: Date.now
+    }
+});
+
+
+// Create the Lesson model
+const adaptiveLearning = mongoose.model("adaptive_learning", adaptive_learning);
```

Comment:
change to learnerai_adaptive_learning_enabled

Reply:
This is not a good practice to give to much bit table name.